editor/src/core/shared/element-path.ts

```diff
@@ -271,6 +271,10 @@ export function parentPath(path: ElementPath): ElementPath {
   }
 }
 
+export function isParentComponentOf(maybeParent: ElementPath, maybeChild: ElementPath): boolean {
```

Comment:
I don't remember why I made this function. should I just use `isParentOf`?

Reply:
It _looks_ equivalent to `isParentOf`.